test/test_pipeline/components/preprocessing/test_imputers.py

```diff
@@ -39,14 +39,14 @@ def test_get_config_space(self):
             self.assertEqual(param1, param2)
 
     def test_mean_imputation(self):
-        data = np.array([['1.0', np.nan, 3],
+        data = np.array([[1.0, np.nan, 3],
                          [np.nan, 8, 9],
-                         ['4.0', 5, np.nan],
+                         [4.0, 5, np.nan],
                          [np.nan, 2, 3],
-                         ['7.0', np.nan, 9],
-                         ['4.0', np.nan, np.nan]], dtype=object)
-        numerical_columns = [1, 2]
-        categorical_columns = [0]
+                         [7.0, np.nan, 9],
+                         [4.0, np.nan, np.nan]])
+        numerical_columns = [0, 1, 2]
+        categorical_columns = []
```

Comment:
```suggestion
        categorical_columns = []  # This imputation is not for categoricals!    
```

Reply:
Actually none of the imputations are for categorical columns, that is the point of this PR